### ISSUE TITLE
Fix school overview precomputing with district-level admins

### ISSUE THREAD

**Issue by kevinrobinson** [studentinsights/studentinsights#677]
This is only affecting users loading the Brown page at the moment, but we should simplify this across the board (`schools_controller#show`, `PrecomputeStudentHashesJob`, and `students_for_school_overview`.

**Comment by alexsoble** [comment]
Hey @kevinrobinson! If/when you get a chance, could you say more about what you meant here?

**Comment by kevinrobinson** [comment]
@alexsoble Sure!  My bad on this issue, hard to understand exactly what this was about.

If I remember right, when we first did the work to expand to schools beyond the Healey there was an issue with how we computed what students district-level admins had access to, and this has long since been addressed with changes in methods like https://github.com/studentinsights/studentinsights/blob/master/app/models/educator.rb#L63 and https://github.com/studentinsights/studentinsights/blob/master/app/jobs/precompute_student_hashes_job.rb#L27.

I think this is safe to close as completed, since I don't think there are any outstanding issues we know of here anymore.
